app/install/kubernetes.md

```diff
@@ -81,17 +80,18 @@ Following are the steps to install Kong using the manifest files:
     $ kubectl create -f postgres.yaml
     ```
 
+    **Kong Enterprise Edition trial users** should complete the steps in the 
+    **Additional Steps for Kong EE Trial Users** section below before proceeding.
+
 4. **Prepare database**
     
-    **Kong Enterprise Edition trial users** should complete the steps in the **Additional steps for Kong EE Trial users** section below before proceeding.
-
-    Using the `kong_migration_<postgres|cassandra>.yaml` file from this repo,
-    run the migration job, jump to step 5 if Kong migrations are up–to–date:
+    Using the `kong_migration_<postgres|cassandra>.yaml` file,
+    run the migration:
     
     ```bash
     $ kubectl create -f kong_migration_<postgres|cassandra>.yaml
     ```
-    Once job completes, you can remove the pod by running following command:
+    Once the migration completes, you can remove the pod by running following command:
```

Comment:
Job is correct here. It's the Kubernetes object used to implement migrations: https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/

Reply:
Oh nice catch, thanks @rainest !